docs/fsharp/tutorials/type-providers/creating-a-type-provider.md

```diff
@@ -3,6 +3,7 @@ title: "Tutorial: Create a Type Provider"
 description: Learn how to create your own F# type providers in F# 3.0 by examining several simple type providers to illustrate the basic concepts.
 ms.date: 05/16/2016
```

Comment:
Since this has quite a few changes, could you update the `ms.date` field to be today?

Reply:
Done